ecosystem/indexer-grpc/indexer-grpc-data-service/src/metrics.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_metrics_core::{
+    register_gauge_vec, register_histogram_vec, register_int_counter_vec, register_int_gauge_vec,
+    GaugeVec, HistogramVec, IntCounterVec, IntGaugeVec,
+};
+use once_cell::sync::Lazy;
+
+/// Latest processed transaction version.
+pub static LATEST_PROCESSED_VERSION: Lazy<IntGaugeVec> = Lazy::new(|| {
+    register_int_gauge_vec!(
+        "indexer_grpc_data_service_latest_processed_version",
+        "Latest processed transaction version",
+        &["request_token", "processor_name"],
+    )
+    .unwrap()
+});
+
+/// Observed latest processed transaction version.
+pub static OBSERVED_LATEST_PROCESSED_VERSION: Lazy<IntGaugeVec> = Lazy::new(|| {
+    register_int_gauge_vec!(
+        "indexer_grpc_data_service_observed_latest_processed_version",
+        "Latest processed transaction version",
+        &["request_token", "processor_name"],
+    )
+    .unwrap()
+});
+
+/// Number of transactions that served by data service.
+pub static PROCESSED_VERSIONS_COUNT: Lazy<IntCounterVec> = Lazy::new(|| {
+    register_int_counter_vec!(
+        "indexer_grpc_data_service_processed_versions",
```

Comment:
```suggestion
        "indexer_grpc_data_service_processed_versions_count",
```

and same for the others

Reply:
done. i think the `error_count` is the problem after reading this https://github.com/prometheus/docs/blob/main/content/docs/practices/naming.md

changed all `error_count` to `errors`